### PR TITLE
fix: Adjust vsix installer when 2022 is installed

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): fixes https://github.com/unoplatform/uno/issues/6386

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Workaround version selection for vsix when VS2022 is installed

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by nickrandolph — APPROVED** [review]
Looks good

**Comment by PlatinumLucario** [comment]
I can confirm that this doesn't fix the issue, it still comes up with the same error as before, and there may be a chance that it might be missing something

If anyone is to re-replicate this issue, make sure the 2022 preview version is installed with it using the preview version of the Visual Studio Installer is being used

Visual Studio seems to repoint any VSIXInstaller (regardless of location) to wherever the main one is installed

File: src/SolutionTemplate/UnoSolutionTemplate.VISX/source.extension.vsixmanifest [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <Icon>logo.png</Icon>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0]" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
```
Comment (ajpinedam) [inline src/SolutionTemplate/UnoSolutionTemplate.VISX/source.extension.vsixmanifest:12]: This "trick" is new to me :)

**Review by ajpinedam — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.8

**Pull request merged**